### ISSUE TITLE
Cursor blinking under popups

### ISSUE THREAD

**Issue by MrAlexLau** [froala/wysiwyg-editor#407]
I've only noticed this in IE and Safari on IOS. If you have your cursor blinking within the wysiwyg editor, then display the popup for either inserting a video or an image, you can still see the cursor blinking in the background:
![screen shot 2015-01-23 at 2 17 20 pm](https://cloud.githubusercontent.com/assets/75554/5882467/479aa968-a30f-11e4-80ce-030d9c944075.png)

I would expect that clicking on the video and image buttons would cause the editable area of content to lose focus.

**Comment by stefanneculai** [comment]
What IE version are you using?

**Comment by MrAlexLau** [comment]
Tried with IE9, 10, and 11. IOS was on 8.1 (using the simulator)

**Comment by stefanneculai** [comment]
We have investigated this and it is a problem that Microsoft and Apple should fix. 

There is already an issue opened for IE on https://connect.microsoft.com/IE/feedback/details/841043/blinking-text-cursor-overlapping-with-div. Most probably it will be fixed in the next version of IE according to the feedback.

For iOS, we have submitted this issue to Apple and we are currently waiting for feedback.
